Source/WebCore/rendering/RenderTreeAsText.cpp

```diff
@@ -364,58 +364,35 @@ void RenderTreeAsText::writeRenderObject(TextStream& ts, const RenderObject& o,
         if (borderTop || borderRight || borderBottom || borderLeft) {
             ts << " [border:";
 
+            auto printBorder = [&ts, &o] (LayoutUnit const& width, BorderStyle const& style, Color color)
+                {
+                    if (!width)
+                        ts << " none";
+                    else {
+                        ts << " (" << width << "px ";
+                        printBorderStyle(ts, style);
+                        if (!color.isValid())
+                            color = o.style().color();
+                        ts << serializationForRenderTreeAsText(color) << ")";
+                    }
+                };
```

Comment:
To make stylebot happy:

```suggestion
            auto printBorder = [&ts, &o] (LayoutUnit const& width, BorderStyle const& style, Color color) {
                if (!width)
                    ts << " none";
                else {
                    ts << " (" << width << "px ";
                    printBorderStyle(ts, style);
                    if (!color.isValid())
                        color = o.style().color();
                    ts << serializationForRenderTreeAsText(color) << ")";
                }
            };
```